models/public/shufflenet-v2-x1.0/model.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Shufflenet V2 x1.0 is image classification model pretrained on ImageNet dataset.
+  This is PyTorch implementation based on architecture described in paper "ShuffleNet
+  V2: Practical Guidelines for Efficient CNN Architecture Design" <https://arxiv.org/pdf/1807.11164.pdf>
+  in TorchVision package (see here <https://github.com/pytorch/vision>).
+
+  The model input is a blob that consists of a single image of "1x3x224x224" in RGB
+  order.
+
+  The model output is typical object classifier for the 1000 different classifications
+  matching with those in the ImageNet database.
+
+task_type: classification
+files:
+  - name: shufflenetv2_x1-5666bf0f80.pth
+    sha256: 5666bf0f80d1e7f8fb61f6a65742b9c27c37f3208c1f087cdcd7c1ec1aa8b848
+    size: 9218294
+    source: https://download.pytorch.org/models/shufflenetv2_x1-5666bf0f80.pth
+  - name: model/shufflenetv2.py
+    size: 7698
+    sha256: 93f454761eb2cfc91d67271d7af930aa96d43afd10ccc0e1a653c0ad282bac7d
+    source: https://raw.githubusercontent.com/pytorch/vision/227027d5abc8eacb110c93b5b5c2f4ea5dd401d6/torchvision/models/shufflenetv2.py
+  - name: model/utils.py
+    size: 151
+    sha256: cf60594ea5662e1c679307f11b3fb99e5243cf20a247e6f3cc686cad000c9fee
+    source: https://raw.githubusercontent.com/pytorch/vision/227027d5abc8eacb110c93b5b5c2f4ea5dd401d6/torchvision/models/utils.py
```

Comment:
We don't normally download Torchvision source files; we have Torchvision as a dependency of the conversion process, and so we can just use `--import-module=torchvision.models`. Is there any reason not to do that in this case?